mlir/lib/IR/AffineExpr.cpp

```diff
@@ -237,6 +239,135 @@ unsigned AffineDimExpr::getPosition() const {
   return static_cast<ImplType *>(expr)->position;
 }
 
+// Returns true if this expression is made out of only symbols.
+static bool isSymbolic(AffineExpr expr) {
+  if (expr.getKind() == AffineExprKind::SymbolId)
+    return true;
+  return false;
+}
+
+// Returns true if the expression is divisible by the given symbol.
+static bool isSymbolicDivisible(AffineExpr main_expr, unsigned pos) {
```

Comment:
`isDivisibleBySymbol`

`pos` -> `symbolPos` 

Pretty important to choose the right names for readability.

Reply:
main_expr: please use camel back. Just expr would be fine.